dashboard/db/migrate/20210212001501_create_frameworks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFrameworks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :frameworks do |t|
```

Comment:
For these migrations that create tables it might be worth it to explicitly set the character set to utf8 like [this migration does](https://github.com/code-dot-org/code-dot-org/blob/ad4d6c0dbe1a495d1116041fde25c871c86e9b7f/dashboard/db/migrate/20210126021131_add_topic_to_featured_projects.rb#L5) to avoid some of the issues we've seen lately between staging and test.

Reply:
@bethanyaconnor I like the idea of making migrations utf8mb4-proof. when I added this and then tried to rollback my migrations, I got:
```
This migration uses execute, which is not automatically reversible.
To make the migration reversible you can either:
1. Define #up and #down methods in place of the #change method.
2. Use the #reversible method to define reversible behavior.
```
I didn't want to switch to defining #up and #down, but `reversible` was pretty easy to add.

Reply:
Ah makes sense. Thanks for finding a solution!